idmega2000/visitrail
# #167382838 Company admin add users

**What does this PR do?**
It implements the ability for a company admin to add users

**Description of Task to be completed?**
- create company user view
- create company user serializer
- implement company user add user
Describe (in detail) what the task completed in the pull request does as per stated in the
pivotal tracker story

**How should this be manually tested?**
- Fetch the branch `ft-company-admin-add-user-167382838`
- Check out into the branch `ft-company-admin-add-user-167382838`
- run python manage.py migrate
- run python manage.py runserver
- make a post request to `http://localhost:8000/api/v1/companyuser` with the data of the users
- You should get a success message

**Any background context you want to provide?**

N/A
**What are the relevant pivotal tracker stories?**
[#167382838](https://www.pivotaltracker.com/story/show/167382838)
